src/libraries/System.Security.Cryptography.X509Certificates/src/System.Security.Cryptography.X509Certificates.csproj

```diff
@@ -14,6 +14,7 @@
          TODO: [AndroidCrypto] Remove ANDROID_OPENSSL_AAR check once Android native crypto is complete -->
     <UseAndroidCrypto Condition="'$(TargetsAndroid)' == 'true' and '$(ANDROID_OPENSSL_AAR)' == ''">true</UseAndroidCrypto>
     <UseAppleCrypto Condition="'$(TargetsOSX)' == 'true' or '$(TargetsiOS)' == 'true' or '$(TargetstvOS)' == 'true'">true</UseAppleCrypto>
+    <ExcludeAppleCryptoOnMobile Condition="'$(TargetsiOS)' == 'true' or '$(TargetstvOS)' == 'true'">true</ExcludeAppleCryptoOnMobile>
```

Comment:
I highly recommend not using "negative named" properties. It's often caused confusion in the past because when you set it to false, there's a double negative.

Consider `<IncludeAppleCryptoOnMobile>`

Reply:
Addressed, thanks.